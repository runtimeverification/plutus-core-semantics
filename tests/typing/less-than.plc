(program 1.0.0 [ [ {(builtin lessThanInteger) (con 1)} (con 1 ! 3)] (con 1 ! 2) ])