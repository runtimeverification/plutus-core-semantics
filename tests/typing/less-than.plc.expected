<generatedTop>
  <k>
    ( all $a ( type ) ( fun $a ( fun $a $a ) ) ) :: ( type )
  </k>
  <env>
    .
  </env>
</generatedTop>
